core/src/main/java/org/apache/accumulo/core/client/admin/InstanceOperations.java

```diff
@@ -195,9 +208,20 @@ Map<String,String> modifyProperties(Consumer<Map<String,String>> mapMutator)
    *        {@code <ip address>:<port>}
    * @return A list of active scans on tablet server.
    */
+  @Deprecated
   List<ActiveScan> getActiveScans(String tserver)
       throws AccumuloException, AccumuloSecurityException;
 
+  /**
+   * List the active scans on a tablet server.
+   *
+   * @param server server type and address
+   * @return A list of active scans on tablet server.
+   * @since 3.0.0
```

Comment:
It would be nice to have something like the following, not sure if the following javadoc syntax is correct.

```suggestion
   * @throws IllegalArgumentException when the type of server is not SCAN_SERVER or TABLET_SERVER
   * @since 3.0.0
```

Reply:
Added in in ea67851.